README.md

```diff
@@ -91,19 +91,26 @@ Table of Contents
     * Full, detailed list - https://aws.amazon.com/free/?awsf.Free%20Tier%20Types=categories%23alwaysfree
 
   * [Microsoft Azure](https://azure.microsoft.com)
-    * App service - 10 web, mobile or API apps
-    * Functions - 1 million requests per month
-    * DevTest Labs - Enable fast, easy, and lean dev-test environments
-    * Active Directory - 500,000 objects
-    * Active Directory B2C - 50,000 monthly stored users
-    * Azure DevOps - 5 active users
-    * Microsoft IoT Hub - 8,000 messages per day
-    * Load Balancer - 1 free public load balanced IP (VIP)
-    * Notification Hubs - 1 million push notifications
-    * Bandwidth - 5GB egress per month
-    * Azure Cosmos DB - 5GB storage and 400 RUs of provisioned throughput for 1 year
-    * [Azure Static Web Apps](https://azure.microsoft.com/pricing/details/app-service/static/?WT.mc_id=free-for-dev-github-wachegha) — Build, deploy and host static apps and serverless functions, with free SSL, Authenticationa/Authorisations and custom domains
-    * Full, detailed list - https://azure.microsoft.com/en-us/free/
+    * [Virtual Machines](https://azure.microsoft.com/services/virtual-machines/?WT.mc_id=freefordev-github-yolasors) - 1 B1S Linux VM, 1 B1S Windows VM
```

Comment:
What are these WT.mc_id url parts? Looks like a webtrends campaign tracking id?

Reply:
This is something we're asked to add to any links redirecting to MS docs. It's used for knowing from which link people ended up on the website, and is a good reminder to come and updates these links in case the location of the docs changes.

Reply:
Fair enough, but I wont be adding them here sorry.

Reply:
I removed all the WT tags, hope that's ok.